docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2807,6 +2807,284 @@ have not been included. These are:
+will be determined at the character level as XX errors per character. We think this approximates to 1 
+error per selected string, but obviously will not be exact. To implement this, select the strings 
+eligible for noise and then apply the OCR noise function to all strings. 
+
+Limitations: 
+
+- The result of having parameters at the character level is that we cannot tell exactly how many data points have noise in them. With the current settings, it is likely similar to the user selected value, but we will always have fewer errors than expected. This could be improved at a later time. 
+
+**Fake Names**
+
+For a variety of reasons, some repondents might choose to use a fake name rather 
+than their real name on official forms. To account for this, first select the sample 
+to have noise added. Then for everyone selected, replace their name with a random 
+selection from the `list of fake names here <https://github.com/ihmeuw/vivarium_research_prl/tree/main/noise>`_. 
+Please note that the list is separated into first and last names. 
+
+Limitations: 
+
+- Many of the fake first names include some information about the simulant (daughter, child f, minor) all specify something about the simulant. We will not try to match this information, which might lead to illogical information (an older man being labeled as 'daughter') but will not impact PRL. 
+- Someone who is likely to use a fake name might well do so across multiple observers. This would likely increase PRL challenges but will not be included here. 
+
+**Nicknames**
+
+Many people choose to use nicknames instead of their "real" names. A common example is an 
+Alexander who chooses to go by Alex. These individuals might write their nicknames on forms 
+which should be recorded. Here is a list of 1080 names and their `relevant nicknames <https://github.com/carltonnorthern/nicknames/blob/master/names.csv>`_. 
+
+Only those simulants with names in the csv above are eligible to recieve a nickname. First, 
+determine who is eligible for a nickname. Then select simulants for noise. Lastly, replace their 
+name with any of the nicknames included in the csv. If there are multiple options, 
+select at random. 
+
+.. note::
+
+  If the process of determining elgibility prior to selecting simulants for noise is challenging, we can work on finding a simpler approach. 
+
+
+**Miswriting** 
+
+For all numeric miswriting, first select the sample to have noise added. 
+Once the sample is selected, change 1 digit of the number at random to any 
+other random number. 
+
+For example: if the correct SSN is 123456789, a miswriting might look like 123416789. 
+Please confirm that the new digit is an actual error. 
+
+Limitations: 
+
+- This might lead to illogical data, especially for age and dates (e.g., a person who's birthday is 12/87/2000). It is more likely that someone lists an incorrect but still possible birthday/age. However, since the main goal is noise for PRL, we think this is still acceptable. 
```

Comment:
I think we need to clarify how this applies to DOB. It seems as though we are thinking of DOB as a string here. If that is the case, we need to specify the format (for each observer? is it always MM/DD/YYYY?). A similar question applies to month/day swapping.

Reply:
I think what you're talking about is what I ran into when trying to add noise to dates for the case study.

The dates are currently stored as pandas datetime objects in the state table and census files. In this format, it is impossible to replace a date with an invalid date because the datatype only supports real dates.

So yes, in the observers, I think we need to specify that the dates are stored as strings so that noise can be added, and we need to specify how the date should be formatted for each observer. For now, I think it will be sufficient to pick a standard format to use for all observers, but eventually, I think we we will want to match the format to whatever format(s) we see in real data.